lib/smart_answer/question/country_select.rb

```diff
@@ -40,7 +40,7 @@ def load_countries
           countries = countries.reject { |c| @exclude_countries.include?(c.slug) }
         end
         if @additional_countries
-          countries = (countries + @additional_countries).sort { |a, b| a.name <=> b.name }
+          countries = (countries + @additional_countries).sort { |a, b| a.name.sub(/^The/, '') <=> b.name }
```

Comment:
Does it matter if this is case sensitive? And do we care about whitespaces?

Reply:
I don't think the case matters, as we are looking only at the beginning of the string. That said, it won't hurt to take care of it just in case. 👍